api/openapi-spec/api.json

```diff
@@ -74,5 +77,18 @@
      }
     }
    }
-  }
+  },
+  "securityDefinitions": {
```

Comment:
So from the list of authenticators I can see, two can be represented in openapi. Token based and http basic (that is actually using the same header as bearer). I've added both. Some authenticators are reduced to bear token, some are not presentable in openapi.